### PR TITLE
chore: add INITIATIVE_PROCESS.md file

### PR DESCRIPTION
**Pull request by lidoravitan**
add documentation for initiative process setup, related to #374 issue.

### CODE REVIEW

**Comment by chowdhurian** [comment]
Closes #374

File: governance/INITIATIVE_PROCESS.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Initiative Setup Process
+
+- [ ] Template for inclusion of the Node.js Code of Conduct (example)
+- [ ] Template for CONTRIBUTING.md (example)
+- [ ] Meeting minutes configuration (if needed – see this for the four kinds of templates)
```
Comment (chowdhurian) [inline governance/INITIATIVE_PROCESS.md]:
```suggestion
- [ ] Meeting minutes configuration (if needed – see [this](https://github.com/nodejs/create-node-meeting-artifacts/tree/master/templates) for the four kinds of templates)
```

The link was broken.

File: governance/INITIATIVE_PROCESS.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Initiative Setup Process
+
+- [ ] Template for inclusion of the Node.js Code of Conduct (example)
```
Comment (chowdhurian) [inline governance/INITIATIVE_PROCESS.md]:
```suggestion
- [ ] Template for inclusion of the Node.js Code of Conduct ([example](https://github.com/nodejs/community-committee/blob/master/CODE_OF_CONDUCT.md))
```

Link broken.

File: governance/INITIATIVE_PROCESS.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+- [ ] Template for CONTRIBUTING.md (example)
+- [ ] Meeting minutes configuration (if needed – see this for the four kinds of templates)
+- [ ] Add to the STRATEGIC-INITIATIVES.md list
```
Comment (chowdhurian) [inline governance/INITIATIVE_PROCESS.md]:
```suggestion
- [ ] Add to the [STRATEGIC-INITIATIVES.md](https://github.com/nodejs/community-committee#strategic-initiatives) list
```

File: governance/INITIATIVE_PROCESS.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Initiative Setup Process
+
+- [ ] Template for inclusion of the Node.js Code of Conduct (example)
+- [ ] Template for CONTRIBUTING.md (example)
```
Comment (chowdhurian) [inline governance/INITIATIVE_PROCESS.md]:
```suggestion
- [ ] Template for CONTRIBUTING.md ([example](https://github.com/nodejs/i18n/blob/master/CONTRIBUTING.md))
```

**Comment by lidoravitan** [comment]
@chowdhurian thanks for the review, your comments are very clear. can you review again?

**Comment by chowdhurian** [comment]
Thanks for putting time and effort into the PR @lidoravitan 🏆

**Comment by lidoravitan** [comment]
@chowdhurian Thank you for your support/mentoring it was very helpful, Hope to become more and more involved. 💚 

actually, maybe I can open PR about "linking to this file from the README" if it necessary.

**Comment by chowdhurian** [comment]
Could be very helpful to link it from there 💯

**Comment by chowdhurian** [comment]
Could be very helpful to link it from there 💯

**Pull request merged**